src/Files.App/Helpers/RecycleBinHelpers.cs

```diff
@@ -109,8 +109,7 @@ public static async Task RestoreRecycleBin(IShellPage associatedInstance)
 
 			if (result == ContentDialogResult.Primary)
 			{
-				associatedInstance.SlimContentPage.ItemManipulationModel.SelectAllItems();
-				await RestoreItem(associatedInstance);
+				Vanara.Windows.Shell.RecycleBin.RestoreAll();
```

Comment:
Can you use CsWin32 instead of Vanara?

Reply:
> Can you use CsWin32 instead of Vanara?

When you say CsWin32, what DLL/namespace are you referring to? I found a project named CsWin32 on Github by Microsoft, but I reckon that's not it?

Reply:
https://github.com/microsoft/CsWin32

Reply:
@onein528 you seem to know how CsWin32 works, would you happen to know if there is a way to restore the recycle bin from it?

Reply:
No. I didn’t get how that works about string handling. They dont have StringBuilder and have PWSTR. And I couldn’t understand how can I assign them.